modular_skyrat/modules/advanced_shuttles/code/shuttles.dm

```diff
@@ -63,6 +64,22 @@
 	. = ..()
 	return waiting ? "RTN" : .
 
+/**
+ * Checks if our shuttle is occupied by someone or not. Ignores camera eyes and ghosts.
+*/
+/obj/docking_port/mobile/arrivals_skyrat/proc/check_occupied()
+	for(var/P in GLOB.player_list)
```

Comment:
This should probably use `GLOB.alive_player_list`, since it'd probably have to iterate through a lot less entries.